tools/build/test_php_static_check.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+set -xe -o pipefail
+
+show_help() {
```

Comment:
I would follow the way we have in tools/build/build_php_deps.sh --php_versions '81 82 83 84' and iterate through versions in one script call.
Then we can pass versions from properties in workflow and we don't need  remember to update many files when adding support for next php release (or deprecate).

take a look into [build-php-deps.yml](https://github.com/elastic/elastic-otel-php/blob/fbf71a5b48f397809af06476544306af606973cb/.github/workflows/build-php-deps.yml#L24)